config/backend-config/backend-config.go

```diff
@@ -223,10 +224,12 @@ func configUpdate(statConfigBackendError stats.RudderStats) {
 		filteredSourcesJSON := filterProcessorEnabledDestinations(sourceJSON)
 		curSourceJSON = sourceJSON
 		curSourceJSONLock.Unlock()
+		initializedLock.Lock()
```

Comment:
Let's have `defered initializedLock.Unlock()` here to make it more readable?

Reply:
Done